src/app/app.module.ts

```diff
@@ -21,10 +23,12 @@ const routes: Routes = [
     BrowserModule,
     RouterModule.forRoot(routes, { useHash: true }),
     HttpClientModule,
-    NovoElementsModule
+    NovoElementsModule,
+    BrowserAnimationsModule
   ],
   providers: [
-    AppBridgeService
+    AppBridgeService,
+    SettingsService
```

Comment:
Spoke with Tomas. We do not need the SettingsService included in the Open Source Repo